### ISSUE TITLE
【Question】Training loss appears negative

### ISSUE THREAD

**Issue by kevinchiu19** [facebookresearch/vggt#246]
Thank you very much for your open source training code!

When I reproduced the current training code, if I started training from scratch, I found that the current loss design of depth and pointmap would cause negative numbers to appear during training.

<img width="1529" height="79" alt="Image" src="https://github.com/user-attachments/assets/fec05c97-9805-43c1-afa7-4b1ab1fd9e1f" />

Is this a normal phenomenon? If negative numbers appear, can the network still converge normally?

Thanks!

**Comment by daiyixiang666** [comment]
Yes, this is a normal phenomenon

**Comment by kevinchiu19** [comment]
> Yes, this is a normal phenomenon

Thank you for your answer, but if I have multiple losses, the total loss will become smaller. Will this not affect the training of other heads?

**Comment by jytime** [comment]
It will not be a problem

**Comment by kevinchiu19** [comment]
ok, thanks!
